int32 episode_number
float32 episode_reward
int32 total_episode_steps
